src/gen_cranberry/gc_board.h

```diff
@@ -18,7 +20,7 @@ struct gc_packet{
     uint16_t apogee_w_m2;
     int16_t hih6131_temp_decic; // Temperature (Celsius)
     uint16_t hih6131_humidity_pct;  // Humidity (percentage)
-    uint16_t mpl115a2t1_press_kpa;  // Pressure (kPa)
+    uint32_t mpl115a2t1_press_kpa;  // Pressure (kPa)
```

Comment:
This is named wrong - should be `mpl115a2t1_press_pa` instead of `mpl115a2t1_press_kpa`.